src/validation/xmlValidator.node.ts

```diff
@@ -48,7 +48,8 @@ async function getParser (): Promise<typeof parseXml> {
 
 const xmlParseOptions: Readonly<ParserOptions> = Object.freeze({
   nonet: true,
-  compact: true
+  compact: true,
+  noent: true // prevent https://github.com/CycloneDX/cyclonedx-javascript-library/issues/1061
```

Comment:
noent: true means [substitute](https://gnome.pages.gitlab.gnome.org/libxml2/devhelp/libxml2-parser.html) entities.

Omitting this option is the secure default.

Reply:
exactly.
therefore https://github.com/CycloneDX/cyclonedx-javascript-library/security/advisories/GHSA-38gf-rh2w-gmj7 exists,
and the https://github.com/CycloneDX/cyclonedx-javascript-library/pull/1064 introduced test for this